DepositApi/Models/DepositModel.cs

```diff
@@ -13,5 +14,7 @@ public class DepositModel
         public decimal? Percent { get; set; }
 
         public DateTime Date { get; set; }
+
+        public CalculationType? CalculationType { get; set; }
```

Comment:
Just to clarify: which cases do we have with nullable CalculationType?

Reply:
When CalculationType wasn't sent to API. It is nullable just to make sure that the CalculationType was set, because not nullable enum will be set by the middleware to the default value (0).